barretenberg/cpp/src/barretenberg/flavor/goblin_translator.hpp

```diff
@@ -1205,7 +1205,7 @@ class GoblinTranslator {
     class VerifierCommitments : public AllEntities<Commitment> {
       public:
         VerifierCommitments([[maybe_unused]] std::shared_ptr<VerificationKey> verification_key,
-                            [[maybe_unused]] const BaseTranscript& transcript)
+                            [[maybe_unused]] const std::shared_ptr<BaseTranscript> transcript)
```

Comment:
Just FYI Adam has a PR that rightfully deletes the unneeded `transcript` argument here and in other similar locations so you may get a conflict if that goes in first.